### DISCUSSION TITLE
Error in 0.54.0-beta5

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#770] [General]
Hi,

I noticed an error when using the latest beta. Something to do with the download-quota plugin.

![image](https://github.com/user-attachments/assets/8a649d94-360b-4ca8-b575-de4ca40833e9)

Kind regards,
Henk - https://fileserver.ph5hp.nl

**Comment by rejetto** [comment]
thanks for reporting!
it was an oversight.
it's not related to the beta.
update the plugin please.

**Reply by PH5HP** [reply]
Hi,

Yes, I thought it would be plugin related. TNX.
